EasyProblems/Exercises/UVa815.py

```diff
@@ -0,0 +1,45 @@
+"""
```

Comment:
Having separate folder for Exercises makes sense, but is it compatible with the existing ones/others? If not can you please send another change list to make it uniform?